environment_py3.yml

```diff
@@ -1,15 +1,16 @@
 name: rmg_env
 channels:
+  - pytorch
```

Comment:
It has to be before the defaults channel because the pytorch version in defaults is relatively old. If we were too add pytorch to the RMG channel, then RMG would have to be before defaults, not sure what the preferred option is.

Reply:
Oh actually, I just checked pytorch in default Anaconda and it seems they updated to the current version 5 days ago. I can test if the defaults build works correctly with chemprop and then could remove the pytorch channel completely.

Reply:
Removed pytorch channel and force pushed.